src/pre_process/m_check_ib_patches.fpp

```diff
@@ -227,6 +259,28 @@ contains
 
     end subroutine s_check_cylinder_ib_patch_geometry
 
+    !>  This subroutine verifies that the geometric parameters of
+        !!      the model patch have consistently been inputted by
+        !!      the user.
+        !!  @param patch_id Patch identifier
+    subroutine s_check_model_ib_patch_geometry(patch_id)
```

Comment:
Maybe instead of the routines that say s_xxx_model_xxx, we should replace the terminology 'model' with 'stl'? os this would be `s_check_stl_ib_patch_geometry`

Reply:
if we _actually_ support OBJ we should have some examples with it (it would be great if we did!)